src/main/java/com/xpanxion/java/assignments/student5/Worker5.java

```diff
@@ -52,4 +55,22 @@ public void ex4() {
 
         System.out.println(formatter.format(foodDeptSum));
     }
+
+    public void ex5() {
+        List<Person> people = DataAccess.getPeople().stream().map(person -> {
+                    var ssn = person.getSsn();
+                    var ssnLength = ssn.length();
+                    person.setSsn(ssn.substring(ssnLength-4));
+                    return person;
+                }).collect(Collectors.toList());
+        System.out.println(people);
+    }
+
+    public void ex6() {
+        List<Cat> cats =  DataAccess.getCats()
+                .stream().sorted(Comparator.comparing(Cat::getName))
+                .collect(Collectors.toList());
```

Comment:
FYI - We can just put `.toList()` here, no need for `.collect(Collectors.toList());`